### ISSUE TITLE
Forbid `assert False` and equivalents

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#735]
# Rule request

## Thesis

We need to forbid `assert` with false statements inside: `False`, `None`, `[]`, etc.

So, code:

```python
assert None
assert 0, 'message'
```

Should raise a violation.
Similar to #725 

## Reasoning

This won't work.
